test/unit/discovery_rules_test.rb

```diff
@@ -9,39 +9,39 @@
 
   include CommandTestHelper
 
-  context "ListCommand" do
+  describe "ListCommand" do
```

Comment:
Isn't this a RuboCop rule, or is it related to updating a dependency?

Reply:
Updating the dependency. Apparently, you cannot have a context inside a describe block.